strategy-cache-update-and-refresh/non-controlled.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Cache, update and refresh: non controlled page - ServiceWorker Cookbook</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <style>
+   body {
+     text-align: center;
+   }
+  </style>
+</head>
+<body>
+  <h1>Always synchronized</h1>
+  <img src="./asset" alt="sample asset">
+  <p>This image originates from a non controlled page so, if you reload, it will be always synced with the version in the server.</p>
```

Comment:
comma after 'page', not after 'so'

also same comment re 'synced' as on the previous comment